src/Razor/src/Microsoft.NET.Sdk.Razor/Sdk/Sdk.Razor.StaticAssets.ProjectSystem.props

```diff
@@ -0,0 +1,48 @@
+<!--
+***********************************************************************************************
+Sdk.Razor.StaticAssets.ProjectSystem.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+
+  <!-- Properties for supporting static content -->
+  <PropertyGroup>
+
+    <!-- This is a contract we have with the Web SDK to allow them in the future to import this file
+         instead of the whole SDK if needed.
+     -->
+    <_WebProjectSystemGlobsPropsDefined>true</_WebProjectSystemGlobsPropsDefined>
```

Comment:
Since it is meant to be consumed from outside, it is ideal not to have an underscore in the name.